travis.sh

```diff
@@ -52,8 +52,13 @@ if [ "${CIRCLECI}" != "true" ] ; then
     make -j$N -C ../phobos -f posix.mak MODEL=$MODEL unittest
 fi
 
+QUICK_BUILD=0
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] ; then
+	QUICK_BUILD=1
+fi
+
 # test fewer compiler argument permutations for PRs to reduce CI load
-if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+if [ $QUICK_BUILD -eq 1 ]; then
     make -j$N -C test MODEL=$MODEL
```

Comment:
`QUICK_BUILD` is already `0` and it executes ` make -j2 -C test MODEL=32 'ARGS=-O -inline -release'`.
I tried before using the "quick build" option, but it didn't make a significant build time difference.